docs/analysis/methodology.mdx

```diff
@@ -20,32 +20,39 @@ Use higher zoom levels with caution. They yield higher-resolution results but re
 
-We assume passengers have perfect information about all schedules in the system and that they choose routes to minimize their total travel time. We also assume that passengers can board all vehicles, so we do not account for denied boardings due to vehicles or facilities that are inaccessible to people using wheelchairs, crowding, etc. Some routes may have frequencies/headways defined, but not explicit timetables, which can introduce uncertainty into results. Our routing engine (called R5) mitigates this uncertainty by simulating feasible timetables for all headway-based routes. For example, if the Red Line is set to depart Airport Station every ten minutes, without explicit timetables, one simulated timetable could include departures at 7:00, 7:10, 7:20…, and another simulated timetable could include departures at 7:02, 7:12, 7:22… The number of simulated schedules parameter in the analysis settings determines how many simulated timetables are used, rounding up to an integer number of schedules per minute in the departure window.
+Assuming service operates with perfect adherence to fully specified timetables, the time percentile can be interpreted as how inclined travelers are to adjust their departure time based on transit schedules. Low time percentile values imply travelers are flexible and start their journeys at times that minimize their waiting and overall travel time, which is appropriate for passengers accustomed to coordinating their activities with published transit schedules (e.g. commuter rail riders). Higher time percentile values are appropriate for passengers who expect “turn-up-and-go” service (e.g. users in dense areas with frequent service).
+
+### Networks with frequency-based routes
+
+Some routes may have frequencies/headways defined, but not explicit timetables, which can introduce uncertainty into results. When frequency-based routes are present, travel time distributions contain uncertainty due to a second factor: how the simulated schedules for these routes interact with each other and the fully specified exact timetables. Choosing a lower percentile when frequency-based routes are present thus jointly represents a better departure time choice by the passenger and simulated schedules that yield shorter waiting times. 
+
+Conveyal's routing engine mitigates this uncertainty by simulating feasible timetables for all headway-based routes. For example, if the Red Line is set to depart Airport Station every ten minutes, without explicit timetables, one simulated timetable could include departures at 7:00, 7:10, 7:20…, and another simulated timetable could include departures at 7:02, 7:12, 7:22… The number of simulated schedules parameter in the analysis settings determines how many simulated timetables are used, rounding up to an integer number of schedules per minute in the departure window. For more details on this parameter, see our recent paper (["Half-(head)way there"](https://hdl.handle.net/1721.1/146736)).
```

Comment:
It may be worth mentioning here that if there _isn't_ uncertainty about how it will be operated, they should just use the "exact times" switch or phasing and eliminate it entirely (while also speeding up calculations).